src/plugins/tagfilter/tagfilter-edge-cases-fr.hbs

```diff
@@ -222,20 +249,576 @@
 					{ "selector": ".event-location", "value": "/location" },
 					{ "selector": ".event-language", "value": "/language" }
 				]
-			}'>
-				&lt;template>
-					&lt;li data-wb-tags="">
-						&lt;h3>&lt;/h3>
-						&lt;p>&lt;span class="event-date">&lt;/span> | &lt;span class="event-time">&lt;/span>&lt;/p>
-						&lt;p>&lt;strong>Emplacement:&lt;/strong> &lt;span class="event-location">&lt;/span>, &lt;strong>Langue&lt;/strong>: &lt;span class="event-language">&lt;/span>&lt;/p>
-					&lt;/li>
-				&lt;/template>
-			&lt;/ul>
-			&lt;div class="wb-tagfilter-noresult">
-				&lt;p>Aucun item ne correspond à cette combinaison de filtres.&lt;/p>
-			&lt;/div>
-			&lt;div id="filteredEventsListPagination">&lt;/div>
-		&lt;/div>
-	&lt;/div>
-&lt;/section></code></pre>
+			}'&gt;
+				&lt;template&gt;
+					&lt;li data-wb-tags=""&gt;
+						&lt;h3&gt;&lt;/h3&gt;
+						&lt;p&gt;&lt;span class="event-date"&gt;&lt;/span&gt; | &lt;span class="event-time"&gt;&lt;/span&gt;&lt;/p&gt;
+						&lt;p&gt;&lt;strong&gt;Emplacement:&lt;/strong&gt; &lt;span class="event-location"&gt;&lt;/span&gt;, &lt;strong&gt;Langue&lt;/strong&gt;: &lt;span class="event-language"&gt;&lt;/span&gt;&lt;/p&gt;
+					&lt;/li&gt;
+				&lt;/template&gt;
+			&lt;/ul&gt;
+			&lt;div class="wb-tagfilter-noresult"&gt;
+				&lt;p&gt;Aucun item ne correspond à cette combinaison de filtres.&lt;/p&gt;
+			&lt;/div&gt;
+			&lt;div id="filteredEventsListPagination"&gt;&lt;/div&gt;
+		&lt;/div&gt;
+	&lt;/div&gt;
+&lt;/section&gt;</code></pre>
+</details>
+
+FR EXAMPLE 2 UPDATED:
```

Comment:
@EricDunsworth Is there a need to have this?

Reply:
Nope, will remove it in a moment. Btw thanks for catching it :)!

It was probably some filler content I added while prepping the new example locally... only to overlook it later on :P.